### PR TITLE
Remove duplicated package entry.

### PR DESCRIPTION
**Pull request by dlorenc**
I built a lint rule for this as well over here: https://github.com/wolfi-dev/wolfictl/pull/14

Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

File: rust.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ environment:
       - llvm15-dev
       - clang-15
       - openssl-dev
-      - libssh2-dev
```
Comment (rawlingsj) [inline rust.yaml:36]: was it definitely `libssh2-dev`?  I couldn't see the duplicate

Comment (dlorenc) [reply rust.yaml:36]: Yeah it's here on line 30, just outside the preview.

image

Comment (rawlingsj) [reply rust.yaml:36]: doh I was looking at the whole file using the commit sha - i.e. with the duplicate removed

**Review by rawlingsj — APPROVED** [review]

**Comment by dlorenc** [comment]
Forcing merge since the new check is now failing on other PRs :)

**Pull request merged**